content/blog/2017/12/2017-12-11-auto-convert-freestyle-jenkins-jobs-to-coded-devops-pipeline.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Auto-Convert Freestyle Jenkins Jobs to Coded DevOps Pipeline" 
```

Comment:
Why not just "Auto-Convert Freestyle Jobs to Jenkins Pipeline"? "Coded DevOps Pipeline" just uses a DevOps buzzword, which is not relevant in terms of the article. The only usage of "DevOps" text in the article is "DevOps team" which is also not Pipeline-specific. "Jenkins Pipeline" is a standard term on jenkins.io